### ISSUE TITLE
adjustment to seq.py to allow for choice of output to prores

### ISSUE THREAD

**Issue by raecasey** [kieranjol/IFIscripts#138]
could we make an adjustment to the choice for output so that we can also output directly to pr res, without having to go through the specific folder path for seq2prores?
cheers

**Comment by kieranjol** [comment]
Sure, I can add that no problem. Any other codecs you'd like added, maybe h264?

**Comment by raecasey** [comment]
yes, - that would be super thanks, - with watermark and bitc as default for security. if you can give a choice for not, - that would be great. but not an obvious choice, -= something hidden in the help command. which i can update for you if you wish
cheers
R

**Comment by kieranjol** [comment]
Cool, so the bitc/watermark would only be for h264? I'll start with clean prores as that's a quick fix. The bitc/watermark for h264 will need a little more thinking as I'd like to somehow harness the bitc.py microservice in order to cut down on the work.

**Comment by raecasey** [comment]
maybe leave that so. the mezz is fine for the moment and we can always bitc from that. i'd be a bit concerned being able to go straight from tiff to h264 clean files for security reasons.
cheers

**Comment by kieranjol** [comment]
Prores and command line arguments are added here in this commit. Run update.py to get the latest version.
https://github.com/kieranjol/IFIscripts/commit/a8612998c7e8e1f118ae390e9df3fb53120136b1

The command line argument for prores is `-p`.
There are also some other changes in there too but that's the main one.

**Comment by kieranjol** [comment]
P.S I'm not closing this until we add bitc/h264 support
